src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Encoder.cs

```diff
@@ -70,16 +66,16 @@ public static unsafe OperationStatus EncodeToUtf8(ReadOnlySpan<byte> bytes, Span
                     byte* end = srcMax - 64;
                     if (Vector512.IsHardwareAccelerated && Avx512Vbmi.IsSupported && (end >= src))
                     {
-                        Avx512Encode(ref src, ref dest, end, maxSrcLength, destLength, srcBytes, destBytes);
+                        Avx512Encode<TBase64Encoder, T>(ref src, ref dest, end, maxSrcLength, destLength, srcBytes, destBytes);
 
                         if (src == srcEnd)
                             goto DoneExit;
                     }
 
-                    end = srcMax - 64;
+                    end = srcMax - 32;
```

Comment:
Why did this change from 64 to 32?

Reply:
It was 32 originally, but unintentionally changed with [this PR ](https://github.com/dotnet/runtime/pull/92241)

The method comment also mentions it require 32 byte
https://github.com/dotnet/runtime/blob/e617b8f47390727e6c211797051eac63a70881f5/src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Encoder.cs#L316-L322